ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -590,14 +625,29 @@ export const ProfilePage = fp.flow(
             </div>
             <hr style={{...styles.verticalLine, width: '15.8rem'}}/>
             <div style={{display: 'grid', gap: '10px', gridAutoRows: '225px', gridTemplateColumns: '220px 220px'}}>
+              {controlledTierEnabled && <ProfileRegistrationStepStatus
+                title={<span><i>All of Us</i> Controlled Tier Data Training</span>}
+                wasBypassed={!!controlledTierBypassTime}
+                incompleteButtonText={'Get Started'}
+                completedButtonText={'Completed'}
+                isComplete={!!(controlledTierCompletionTime || controlledTierBypassTime)}
+                // TODO: link to the training modules once they are available
+                completeStep={() => null}
+                content={getControlledTierContent({controlledTierCompletionTime, controlledTierBypassTime})}
+                >
+                <div>
```

Comment:
Based on the previous change:
The child tags will now always render in the profile card. The items in the `content` prop will render conditionally.